Makefile

```diff
@@ -162,14 +163,12 @@ build-ia-operator-reference:
 			--env SSH_AUTH_SOCK=/ssh-agent\
 			-w /go/src/$(PACKAGE_NAME) \
 			$(CALICO_BUILD) /bin/bash -c '$(GIT_CONFIG_SSH) rm -rf builder && mkdir builder && cd builder && \
-				git clone --depth=1 -b ia_kb_v2 https://github.com/Brian-McM/gen-crd-api-reference-docs api-gen && cd api-gen && \
-				go mod edit -replace github.com/tigera/image-assurance=github.com/$(IA_OPERATOR_REPO)@$(IA_OPERATOR_VERSION) && \
-				go mod tidy && go mod download all && go build && \
-				./gen-crd-api-reference-docs \
-					-api-dir github.com/tigera/image-assurance/operator/api \
+				git clone --depth=1 -b $(API_GEN_VERSION) git@github.com:ahmetb/gen-crd-api-reference-docs.git && cd gen-crd-api-reference-docs && go build && \
```

Comment:
I understand which we need a separate section here just for IA operator, but it feels like we should at least reconcile with the one that is being used for the Operator: 

- https://github.com/tigera/docs/blob/main/Makefile#L27-L28
- https://github.com/tigera/docs/blob/main/Makefile#L141

Reply:
I'll leave that up to @tmjd since we were all using custom forks previously. Likely we should just use the base repo for everything, but I'm not invested into updating the tigera operator API docs right now for cloud.